src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2936,6 +2919,32 @@ public void MakeNewHomeWorkspace(object parameter)
 
         private void CloseHomeWorkspace(object parameter)
         {
+            // Upon closing a workspace, validate if the workspace is valid to be sent to the ML datapipeline and then send it.
+            if (!DynamoModel.IsTestMode && !HomeSpace.HasUnsavedChanges && (currentWorkspaceViewModel?.IsHomeSpace ?? true) && HomeSpace.HasRunWithoutCrash)
+            {
+                if (!IsMLDataIngestionPipelineinBeta && Model.CurrentWorkspace.IsValidForFDX && currentWorkspaceViewModel.Checksum != string.Empty)
```

Comment:
This would be disabled by default correct with the value being `true`?

Reply:
Yes will be disabled with the flag value true. We will be switching the flag value to false once the release is done?

Reply:
@reddyashish Yes, once we know it is integrated into Revit for sure